modules/saliency/include/opencv2/saliency/saliencySpecializedClasses.hpp

```diff
@@ -405,6 +405,173 @@ class CV_EXPORTS ObjectnessBING : public Objectness
 
 //! @}
 
+
+
+/**
+ * \brief Objectness algorithms based on [4]
+ * [4] Zitnick, C. Lawrence, and Piotr Dollár. "Edge boxes: Locating object proposals from edges." Computer Vision–ECCV 2014
+ */
+
+/** @brief the EdgeBoxes algorithm from @cite EdgeBoxes
+ */
+class CV_EXPORTS ObjectnessEdgeBoxes : public Objectness
+{
+public:
+
+	const int EDGE_IMAGE = 0;
```

Comment:
@ultrafro, we are testing OpenCV without C++11 support and these constructions does not work in older standard. It seems that these constants are not used anywhere, so the can be deleted.